README.md

```diff
@@ -19,7 +19,7 @@ about the currently available features, limitations and known bugs.
 
 ### Features
 
-- Cross-platform (Unix/Linux, Mac OS X, Windows)
+- Cross-platform (Unix/Linux/FreeBSD, Mac OS X, Windows)
```

Comment:
The compatibility is not specific to FreeBSD, right?

Also, if you're at it, you could fix the macOS name :slightly_smiling_face: 

```suggestion
- Cross-platform (Unix/Linux/BSD, macOS, Windows)
```

Reply:
Since there is no reaction on this suggestion, I'll apply it by myself ;)

Thanks @yurivict for updating the README!

Reply:
Hmm it seems I have no permissions to push to this branch, so I can't apply the suggestion.

@yurivict could you either apply the suggestion or grant me write permissions on this branch?